src/containers/ServiceWorkerFallbackPage.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+import React, { Component } from 'react'
+import get from 'lodash/get'
+import styled from 'styled-components'
+import withLayout from '../helpers/with-layout'
+import { Link, withRouter } from 'react-router'
+import { connect } from 'react-redux'
+import { push } from 'react-router-redux'
+
+const _ = {
+  get
+}
+
+const Container = styled.div`
+  max-width: 95%;
+  margin: 0 auto;
+  height: 100vh;
+  display: flex;
+`
+
+const CenteredBox = styled.div`
+  margin: auto;
+`
+
+const P = styled.p`
+  line-height: 1.8;
+  a {
+    color: red;
+    border-bottom: 1px solid red;
+  }
+`
+
+class FallbackPage extends Component {
+  //componentDidMount() {
+  //  const pathname = _.get(this.props, 'location.pathname')
+  //  console.error('pathname:', pathname)
+  //  if (pathname !== '/sw-fallback-page') {
+  //    this.props.push(`/sw-fallback-page?redirect=${pathname}`)
+  //  }
+  //}
```

Comment:
Is there any situation we need to push new location? or can this be removed?

Reply:
it can be removed.